store.html

```diff
@@ -0,0 +1,119 @@
+                    </div>
+                </div>
+                <div class="form-line">
+                    <div class="form-items-holder">
+                 <label for="address2" class="form-input">Address 2</label>
+                 <input id="address2" type="text" name="address2" class="address">
+                    </div>
+                </div>
+                <div class="form-line">
+                    <div class="form-items-holder">
+                 <label for="city" class="form-input">City *</label>
+                 <select id="city" name="city" required>
+                    <option value="-1" selected>Select your city...</option>
+                    <option value="london">London</option>
+                    <option value="milan">Milan</option>
+                    <option value="birmingham">Birmingham</option>
+                    <option value="paris">Paris</option>
+                 </select>
+                    </div>
+                    <div class="store-free-space"></div>
+                    <div class="form-items-holder">
+                <label for="postcode" class="form-input">Postcode *</label>
+                <input id="postcode" type="text" name="postcode" maxlength="8" style="text-transform:uppercase">
+                    </div>
+                </div>
+                <div class="form-line">
+                     <fieldset class="form-input" id="select-colour">
+                        <legend>Select a colour</legend>
+                      <div class="radio-btns">
+                          <div class="radio-lebel-holder">
+                            <input id="orange" type="radio" name="orange" value="orange">
+                            <label for="orange" class="radio-label">Karma orange</label>
+                          </div>
+                          <div class="radio-lebel-holder">
+                           <input id="grey" type="radio" name="orange" value="grey">
+                           <label for="grey" class="radio-label">Space Grey</label>
+                          </div>
+                     </div>
+                    </fieldset>
+                </div>
+                <div class="form-line" id="checkbox-items">
+                 <p class="terms"> <input type="checkbox" class="checkbox-btn" required>By placing your order you agree to Karma's <span style="color:#e0633a">Terms and</span><br><span style="color:#e0633a">Conditions.</span> *</p>
+                </div>
+                <div class="form-line" id="place-order-btn">
+                    <button class="place-order-btn">Place my order</button>
+                </div>
+            </form>
+        </section>
+        <img src="img/store-image_by-andrew-neel-unsplash.jpg" alt="store-image" class="store-img">
+    </main>
+    <footer class="footer">
+        <div class="footer-line"></div>
+        <h2 class="footer-msg">Join us on</h2>
+        <div class="footer-icons">
```

Comment:
This isnt that cruical, on my screen the icons were way to large for the "buttons".